lib/rubocop/cop/style/command_literal.rb

```diff
@@ -0,0 +1,94 @@
+# encoding: utf-8
+
+module RuboCop
+  module Cop
+    module Style
+      # This cop checks for usage of the %x() syntax when `` would do.
+      class CommandLiteral < Cop
+        include ConfigurableEnforcedStyle
+
+        MSG_USE_BACKTICKS = 'Prefer using backticks around command string.'
+        MSG_USE_PERCENT_X = 'Prefer using `%x` around command string.'
+
+        def on_xstr(node)
+          if backtick_literal?(node)
+            check_backtick_liteal(node)
+          elsif percent_x_literal?(node)
+            check_percent_x_literal(node)
+          end
+        end
+
+        private
+
+        def check_backtick_liteal(node)
+          return if style == :backticks && !contains_disallowed_backtick?(node)
+          return if style == :mixed &&
+                    single_line?(node) &&
+                    !contains_disallowed_backtick?(node)
+
+          add_offense(node, :expression, MSG_USE_PERCENT_X)
+        end
+
+        def check_percent_x_literal(node)
+          return if style == :backticks && contains_disallowed_backtick?(node)
+          return if style == :percent_x
+          return if style == :mixed && multi_line?(node)
+          return if style == :mixed && contains_disallowed_backtick?(node)
+
+          add_offense(node, :expression, MSG_USE_BACKTICKS)
+        end
+
+        def contains_disallowed_backtick?(node)
+          !allow_inner_backticks? && contains_backtick?(node)
+        end
+
+        def allow_inner_backticks?
+          cop_config['AllowInnerBackticks']
+        end
+
+        def contains_backtick?(node)
+          node_body(node) =~ /`/
+        end
+
+        def node_body(node)
+          if backtick_literal?(node)
+            node.loc.expression.source[1...-1]
+          elsif percent_x_literal?(node)
+            node.loc.expression.source[3...-1]
+          end
+        end
+
+        def backtick_literal?(node)
+          node.loc.begin.source == '`' && node.loc.end.source == '`'
+        end
+
+        def percent_x_literal?(node)
+          node.loc.begin.source == '%x(' && node.loc.end.source == ')'
+        end
+
+        def single_line?(node)
+          !multi_line?(node)
+        end
+
+        def multi_line?(node)
+          block_length(node) > 1
+        end
+
+        def autocorrect(node)
+          fail CorrectionNotPossible if contains_backtick?(node)
+
+          if backtick_literal?(node)
+            replacement = %w(%x( ))
```

Comment:
You can make an instance of the cop and use its `config` method. As for the tests - you always stub this.